soccerdata/fbref.py

```diff
@@ -120,9 +120,15 @@ def _is_complete(self, league: str, season: str) -> bool:
             return date.today() >= season_ends
         return super()._is_complete(league, season)
 
-    def read_leagues(self) -> pd.DataFrame:
+    def read_leagues(self, optimise_big5: bool = True) -> pd.DataFrame:
         """Retrieve selected leagues from the datasource.
 
+        Parameters
+        ----------
+        optimise_big5: bool
+            If True, it will load the "Big 5 European Leagues Combined" instead of
+            each league individually.
```

Comment:
That's not what it does! Consider the example below. According to the docstring, it should return a data frame with a single row with league="Big 5 European Leagues Combined". Please change to `split_up_big5 = False` as I suggested or change the implementation accordingly.

```py
fbref = sd.FBref(["ENG-Premier League", "FRA-Ligue 1", "ITA-Serie A", "ESP-La Liga", "GER-Bundesliga"], "2223")
fbref.read_leagues(optimise_big5 = True)
```

Reply:
Good catch! I changed the implementation accordingly.